test/plug/html_test.exs

```diff
@@ -10,4 +10,14 @@ defmodule Plug.HTMlTest do
     assert html_escape("\"quoted\"") == "&quot;quoted&quot;"
     assert html_escape("html's test") == "html&#39;s test"
   end
+
+  test "escapes HTML to iodata" do
+    assert iodata_escape("<script>") == "&lt;script&gt;"
+    assert iodata_escape("html&company") == "html&amp;company"
+    assert iodata_escape("\"quoted\"") == "&quot;quoted&quot;"
+    assert iodata_escape("html's test") == "html&#39;s test"
+  end
+
+  defp iodata_escape(data),
+    do: data |> Plug.HTML.html_escape_to_iodata |> IO.iodata_to_binary
```

Comment:
Can we get this without the `, do:` style? I don't think we usually do it for just one-clause functions 🤔